### DISCUSSION TITLE
SIP-0001 discussion

### DISCUSSION THREAD

**Discussion by Semisol** [stackchainsats/stackchain#2] [General]
See: https://github.com/stackchainsats/stackchain/blob/main/sips/sip-0001.txt
Proposer: @derekross 
Twitter thread: https://twitter.com/Semisol_Public/status/1549867225170993152

**Comment by derekross** [comment]
As stated in the abstract, as the initial chain grows in USD amounts, it's harder for plebs with little spare fiat to participate. Creation of a new sidechain would allow for additional plebs to stack. It needs to be done properly to track each new chain. The initial process does allow us to approve and track chains on a case per case basis, always looping in the main chain.

**Comment by flav1000** [comment]
I personally prefer allowing users to stackjoin, this way there's no need to create parallel stackchains.

**Reply by derekross** [reply]
When I wrote SIP-0001 I was not aware that we had the ability to Stackjoin. We should get this written up into a formal SIP so that we have rules defined.

**Comment by derekross** [comment]
The community has rejected this SIP. It will be updated as such. Closing discussion.
